example/android/build.gradle

```diff
@@ -12,7 +12,8 @@ buildscript {
 
         // Hyper SDK
         clientId = 'geddit'
-        hyperSDKVersion = "2.1.15"
+        hyperSDKVersion = "2.1.30"
+        excludedMicroSDKs = []
```

Comment:
This is a very niche use case. We should not put this by default in our demo app